src/window.rs

```diff
@@ -25,6 +25,23 @@ use MouseCursor;
 use cli::Options;
 use config::WindowConfig;
 
+/// This the default value for the `WM_NAME` property.
+///
+/// ```
+/// WM_NAME(STRING) = "Alacritty"
+/// ```
+pub const DEFAULT_TITLE: &str = "Alacritty";
+
+/// This the default value for the `WM_CLASS` property.
+///
+/// The second value of `WM_CLASS` is **never** changed to anything but the
+/// default value.
+///
+/// ```
+/// WM_CLASS(STRING) = "Alacritty", "Alacritty"
+/// ```
+pub const DEFAULT_CLASS: &str = "Alacritty";
+
```

Comment:
These docs for title and class are super X11 specific, however I'm pretty sure at least the title is probably a thing on Wayland and MacOS too.

I don't think a super explicit documentation is required here, since it's not something which is likely going to be used by anyone else. Something like this should be enough:

```rust
/// Text which will be displayed in the window's title bar if it's not overridden by the user
pub const DEFAULT_TITLE: &str = "Alacritty";
```

I'm not sure which platforms the window class is used for, but it looks like it's only X11, XWayland and potentially Wayland? So for this it would probably be fine to be a bit more specific when it comes to documentation, maybe something like this:
```rust
/// General window class for X11, used as an identifier
/// Even if overriden by the user, this is still listed as an alternative in the `WM_CLASS` property
pub const DEFAULT_CLASS: &str = "Alacritty";
```

Reply:
Let's learn more about how this works on other platforms before we do anything then?

Reply:
I don't think what I said is inaccurate. The title should definitely exist on all platforms.

Reply:
It's not innacruate (afaik) but it would be nice to know what to say about macOS for the class was my only point.

Reply:
Added some questions in TODOs to track this.
:checkered_flag: